cmd/api.go

```diff
@@ -26,6 +27,8 @@ func init() {
 	rootCmd.AddCommand(issuance)
 	rootCmd.AddCommand(status)
 	rootCmd.AddCommand(burn)
+	rich.Flags().Int("count", 100, "The top X address")
+	rootCmd.AddCommand(rich)
```

Comment:
I don't have a strong opinion on how we format the cli, but should we nest this under `get`?
E.g
`pegentd get richlist`

I even though about moving the `balances` to the subquery. At some point, we should probably come up with a cli formatting ideology

Reply:
if we clean up the cli structure, let's do it in a separate PR. we can also pull in balance(s), status, and issuance into "get"